From 1c745717ff7d7391266522705143298f3563ad40 Mon Sep 17 00:00:00 2001
From: Daniel Cole <daniel@judgezarbi.com>
Date: Mon, 12 Mar 2018 01:40:38 +0000
Subject: [PATCH] Removed a couple of unintentional bonus characters

---
 librtmp/dh.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/librtmp/dh.h b/librtmp/dh.h
index dcbeead..6440e1c 100644
--- a/librtmp/dh.h
+++ b/librtmp/dh.h
@@ -31,7 +31,7 @@
 #include <polarssl/dhm.h>
 typedef mpi * MP_t;
 #define MP_new(m)	m = malloc(sizeof(mpi)); mpi_init(m)
-=)#define MP_set_w(mpi, w)	mpi_lset(mpi, w)
+#define MP_set_w(mpi, w)	mpi_lset(mpi, w)
 #define MP_cmp(u, v)	mpi_cmp_mpi(u, v)
 #define MP_set(u, v)	mpi_copy(u, v)
 #define MP_sub_w(mpi, w)	mpi_sub_int(mpi, mpi, w)
