Index: rtmpdump-20111228/Makefile
===================================================================
--- rtmpdump-20111228/Makefile
+++ rtmpdump-20111228/Makefile	2011-12-28 21:35:00.152254225 +0100
@@ -25,11 +25,9 @@
 LDFLAGS=-Wall $(XLDFLAGS)
 
 bindir=$(prefix)/bin
-sbindir=$(prefix)/sbin
 mandir=$(prefix)/man
 
 BINDIR=$(DESTDIR)$(bindir)
-SBINDIR=$(DESTDIR)$(sbindir)
 MANDIR=$(DESTDIR)$(mandir)
 
 LIBS_posix=
@@ -59,9 +57,8 @@
 $(PROGS): $(LIBRTMP)
 
 install:	$(PROGS)
-	-mkdir -p $(BINDIR) $(SBINDIR) $(MANDIR)/man1 $(MANDIR)/man8
-	cp rtmpdump$(EXT) $(BINDIR)
-	cp rtmpgw$(EXT) rtmpsrv$(EXT) rtmpsuck$(EXT) $(SBINDIR)
+	-mkdir -p $(BINDIR) $(MANDIR)/man1 $(MANDIR)/man8
+	cp rtmpdump$(EXT) rtmpgw$(EXT) rtmpsrv$(EXT) rtmpsuck$(EXT) $(BINDIR)
 	cp rtmpdump.1 $(MANDIR)/man1
 	cp rtmpgw.8 $(MANDIR)/man8
 	@cd librtmp; $(MAKE) install
