diff -Nurpa -x '*~' -x '*.orig' -x '*.rej' -x '*.swp' rtmpdump-2.3//Makefile newdir/Makefile
--- rtmpdump-2.3//Makefile	2010-07-20 00:02:29.252541749 +0300
+++ rtmpdump-2.3/Makefile	2010-07-20 00:02:42.046624140 +0300
@@ -51,7 +51,7 @@ all:	$(LIBRTMP) $(PROGS)
 
 $(PROGS): $(LIBRTMP)
 
-install:	progs
+install:	$(PROGS)
 	-mkdir -p $(BINDIR) $(MANDIR)/man1 $(MANDIR)/man8
 	cp rtmpdump$(EXT) rtmpgw$(EXT) rtmpsrv$(EXT) rtmpsuck$(EXT) $(BINDIR)
 	cp rtmpdump.1 $(MANDIR)/man1
